content/docs/2.10/scalers/prometheus.md

```diff
@@ -22,7 +22,8 @@ triggers:
     activationThreshold: '5.5'
     # Optional fields:
     namespace: example-namespace  # for namespaced queries, eg. Thanos
-    cortexOrgID: my-org # Optional. X-Scope-OrgID header for Cortex.
+    cortexOrgID: my-org # Deprecated: Use custom headers instead to set X-Scope-OrgID header for Cortex. (see below)
```

Comment:
Can you please align with this pattern here + in the list below?

`# DEPRECATED: This parameter is deprecated as of KEDA v2.10 in favor of {replacement} and will be removed in version 2.12`